### PR TITLE
building warnings reduced

### PR DESCRIPTION
**Pull request by Dorbedo**
**When merged this pull request will:**
- reduces the compiling warnings with common tools
- could use some test with the current release creating to make sure I didn't break too much

### CODE REVIEW

File: addons/backpacks/anarc164/ui/anarc164.ext [hunk 1]
```diff
@@ -91,17 +91,7 @@ class anarc164_radio_dialog
...
-            _f = TFAR_FREQUENCYSTRING_TO_FREQNUMBER(ctrlText IDC_ANARC164_EDIT);
-            if ((_f >= TFAR_MIN_ASIP_FREQ) and (_f <= TFAR_MAX_ASIP_FREQ)) then {
-                [ARR_2(TF_lr_dialog_radio, QTFAR_ROUND_FREQUENCY(_f))] call TFAR_fnc_setLrFrequency;
-                call TFAR_fnc_hideHint;
-            } else {
-                hint formatText [ARR_3(localize 'STR_TFAR_CORE_incorrect_frequency', TFAR_MIN_ASIP_FREQ, TFAR_MAX_ASIP_FREQ)]
-            };
-            ["%1"] call TFAR_fnc_updateLRDialogToChannel;
-          );
+        action = QUOTE(playSound 'TFAR_rotatorPush';_f = TFAR_CONFIG_FREQUENCYSTRING_TO_FREQNUMBER(ctrlText IDC_ANARC164_EDIT);if ((_f >= TFAR_MIN_ASIP_FREQ) and (_f <= TFAR_MAX_ASIP_FREQ)) then {[ARR_2(TF_lr_dialog_radio, str TFAR_ROUND_FREQUENCY(_f))] call TFAR_fnc_setLrFrequency;call TFAR_fnc_hideHint;} else {hint formatText [ARR_3(localize 'STR_TFAR_CORE_incorrect_frequency', TFAR_MIN_ASIP_FREQ, TFAR_MAX_ASIP_FREQ)];};['%1'] call TFAR_fnc_updateLRDialogToChannel;);
```
Comment (dedmen) [inline addons/backpacks/anarc164/ui/anarc164.ext]: This is terrible though. What warnings did you get here?
First thing I see tho Line 103 had " which might be bad-ish with QUOTE.

File: addons/handhelds/radio_ids.hpp [hunk 2]
```diff
@@ -1,73 +1,1012 @@
...
-        displayName = displayNameBase index; \
+        displayName = 'displayNameBase index'; \
         scope = 1; \
         scopeCurator = 1; \
         tf_prototype = 0; \
         tf_radio = 1; \
         ace_arsenal_uniqueBase = baseClass; \
     };
 
+#define TF_RADIO_ID_LIST(baseClass,displayNameBase) \
```
Comment (dedmen) [inline addons/handhelds/radio_ids.hpp]: No. What tools are you using that are too dumb for that?

Comment (Dorbedo) [reply addons/handhelds/radio_ids.hpp]: Mikereo.... I was surprised too, but it's the way it is....

Comment (dedmen) [reply addons/handhelds/radio_ids.hpp]: We use Mikero on CI and it works fine

**Pull request closed without merging**